apiservers/engine/backends/container.go

```diff
@@ -173,22 +174,41 @@ func (c *Container) ContainerCreate(config types.ContainerCreateConfig) (types.C
 	}
 
 	plCreateParams := c.dockerContainerCreateParamsToPortlayer(config, layer.ID, host)
-	createResults, err := client.Exec.ContainerCreate(plCreateParams)
-
+	createResults, err := client.Containers.Create(plCreateParams)
 	// transfer port layer swagger based response to Docker backend data structs and return to the REST front-end
 	if err != nil {
-		if _, isa := err.(*exec.ContainerCreateNotFound); isa {
+		if _, isa := err.(*containers.CreateNotFound); isa {
 			return types.ContainerCreateResponse{}, derr.NewRequestNotFoundError(fmt.Errorf("No such image: %s", layer.ID))
 		}
 
 		// If we get here, most likely something went wrong with the port layer API server
-		return types.ContainerCreateResponse{},
-			derr.NewErrorWithStatusCode(fmt.Errorf("Unknown error from the exec port layer"), http.StatusInternalServerError)
+		return types.ContainerCreateResponse{}, derr.NewErrorWithStatusCode(err, http.StatusInternalServerError)
+	}
+
+	id := createResults.Payload.ID
+	h := createResults.Payload.Handle
+
+	// configure networking
+	netConf := toModelsNetworkConfig(config)
+	if netConf != nil {
+		addContRes, err := client.Scopes.AddContainer(
+			scopes.NewAddContainerParams().WithHandle(h).WithNetworkConfig(netConf))
+		if err != nil {
+			return types.ContainerCreateResponse{}, derr.NewErrorWithStatusCode(err, http.StatusInternalServerError)
+		}
+
+		h = addContRes.Payload
+	}
+
+	// commit the create op
+	_, err = client.Containers.Commit(containers.NewCommitParams().WithHandle(h))
+	if err != nil {
+		return types.ContainerCreateResponse{}, derr.NewErrorWithStatusCode(err, http.StatusInternalServerError)
 	}
 
 	// Success!
-	log.Printf("container.ContainerCreate succeeded.  Returning container id %s", *createResults.Payload.ContainerID)
-	return types.ContainerCreateResponse{ID: *createResults.Payload.ContainerID}, nil
+	log.Printf("container.ContainerCreate succeeded.  Returning container handle %s", *createResults.Payload)
```

Comment:
Should this print the container ID, or are you wanting to print the entire Payload struct?